common/src/backends/auth.rs

```diff
@@ -411,7 +411,7 @@ mod tests {
     fn to_token_and_back() {
         let mut claim = Claim::new(
             "ferries".to_string(),
-            vec![Scope::DeploymentRead, Scope::Project],
+            vec![Scope::DeploymentRead, Scope::ProjectRead],
```

Comment:
Hmm, why were these tests updated?

Reply:
Oh, it's because I changed the name of the Scope from `Project` to `ProjectRead` (to be more informative in terms of the naming about what actions are happening within the system, as an effect of the request). However, I've changed the needed scopes of the `subscribe/unsubscribe` RPC to `Admin`.  

Btw, just realized, I am not sure if renaming the scope enum names would be breaking for our existing users. I have to check.